files/project_team/IFCTunnelDep1.3.ifc

```diff
@@ -0,0 +1,65 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'IFCTunnelDep1.3.ifc',
+/* time_stamp */ '2023-02-21T11:22:35',
+/* author */ ('lars'),
+/* organization */ ('Unknown'),
+/* preprocessor_version */ 'GeometryGymIFCcore v0.2.11.0 by Geometry Gym Pty Ltd built 2023-02-17T09:18:34',
+/* originating_system */ 'IFCTunnelDep1 v1.0.0.0',
+/* authorization */ 'None'
+);
+
+FILE_SCHEMA (('IFC4X4_41272F09'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCTUNNEL('0ZhDb1EcrFlB7Pt8ThZRKT',#10,'My Tunnel',$,$,#3,$,$,$,.ROAD.);
+#5= IFCPERSON('lars',$,$,$,$,$,$,$);
+#6= IFCORGANIZATION($,'Unknown',$,$,$);
+#7= IFCPERSONANDORGANIZATION(#5,#6,$);
+#8= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#9= IFCAPPLICATION(#8,' v1.0.0.0','IFCTunnelDep1 v1.0.0.0','IFCTunnelDep1 v1.0.0.0');
+#10= IFCOWNERHISTORY(#7,#9,$,.ADDED.,1676974955,$,$,1676974955);
+#11= IFCPROJECT('27JcVZzyP2Lfos66GaVpVr',#10,'SPRINT 1.2 example with Project Global Positioning',$,$,$,$,(#25),#12);
+#12= IFCUNITASSIGNMENT((#13,#14,#15,#16,#17,#19));
+#13= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#14= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#18= IFCRELAGGREGATES('0mQHVnMprPbfErV6HrUhF2',#10,$,$,#11,(#4));
+#19= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#20= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#21= IFCDIRECTION((0.0,0.0,1.0));
+#22= IFCDIRECTION((1.0,0.0,0.0));
+#23= IFCAXIS2PLACEMENT3D(#20,#21,#22);
+#24= IFCDIRECTION((1.0,0.0,0.0));
+#25= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,0.0000000254,#23,#26);
+#26= IFCDIRECTION((0.0,1.0));
+#27= IFCGEOGRAPHICCRS('EPSG:4326',$,'World Geodetic System 1984 ensemble',$,$,$);
+#28= IFCRIGIDOPERATION(#25,#27,IFCPLANEANGLEMEASURE(24.4769747),IFCPLANEANGLEMEASURE(61.9908225),0.0);
+#29= IFCCLASSIFICATION('Triona AB',$,$,'My classification',$,$,$);
+#30= IFCRELASSOCIATESCLASSIFICATION('3ddD2x5TvBIvBOSwJAT$a4',#10,$,$,(#11),#29);
+#31= IFCBUILTSYSTEM('3fug7PpajBPAqauiUTIxqk',#10,'A system',$,$,.TUNNEL_SUPPORT.,$);
+#32= IFCRELREFERENCEDINSPATIALSTRUCTURE('2X824mwzn24wNfF5xM5M$D',#10,$,$,(#31),#4);
+#33= IFCRELDECLARES('0XxzEMhRjEsx6ATpetrXW7',#10,$,$,#11,(#31,#43,#45));
+#34= IFCDOCUMENTINFORMATION('ISBN: 978-91-7725-946-6','Titel: Krav - VGU, V\X2\00E4\X0\gars och gators utformning',$,'http://trafikverket.diva-portal.org/smash/get/diva2:1621114/FULLTEXT02.pdf',$,$,$,$,$,$,$,$,'application/pdf',$,$,$,$);
```

Comment:
`Editors` field is empty, and thus an actor declaration by the project is missing as well.

Reply:
Added an Editor. I don't quite understand why the IfcRelDeclares check is reporting an error? Currently the following is declared:
```
#33= IFCRELDECLARES('0uB3HDsab5m8q9MfuymxaG',#10,$,$,#11,(#31,#36,#45,#47));
#31= IFCBUILTSYSTEM('2qhchTsgr1_gWIcwPXP_ih',#10,'A system',$,$,.TUNNEL_SUPPORT.,$);
#35= IFCORGANIZATION('http://www.trafikverket.se','Trafikverket',$,$,$);
#36= IFCACTOR('2FCX85XmbF5hu3VP7$qYVD',#10,$,$,$,#35);
#45= IFCARCHELEMENTTYPE('03tHuUU5rC8AvZRAL_MSnn',#10,'Arch-123',$,$,$,$,$,'MyArchType',.USERDEFINED.);
#47= IFCPROPERTYSETTEMPLATE('1HnsV$PyzCJRmVHrw7NENU',#10,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBorehole',(#48));
```

Reply:
Found the error - the checker was set up wrongly.